src/ripple/ledger/impl/ApplyView.cpp

```diff
@@ -0,0 +1,298 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012, 2013 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <BeastConfig.h>
+#include <ripple/ledger/ApplyView.h>
+#include <ripple/basics/contract.h>
+#include <ripple/protocol/Protocol.h>
+#include <cassert>
+
+namespace ripple {
+
+void
+ApplyView::creditHook (
+    AccountID const& from,
+    AccountID const& to,
+    STAmount const& amount,
+    STAmount const& preCreditBalance)
+{
+}
+
+// Called when the owner count changes
+// This is required to support PaymentSandbox
+void
+ApplyView::adjustOwnerCountHook (
```

Comment:
There was no need to put this in the .cpp file it was more concise in the header. This adds additional lines with no corresponding benefit. Same for the other empty function.